openedx/core/djangoapps/theming/core.py

```diff
@@ -47,7 +47,7 @@ def comprehensive_theme_changes(theme_dir):
 
     favicon = component_dir / "static" / "images" / "favicon.ico"
     if favicon.isfile():
-        changes['settings']['FAVICON_PATH'] = str(favicon)
+        changes['settings']['FAVICON_PATH'] = "images/favicon.ico"
```

Comment:
what is reason for changing `favicon` path here whereas availability of favicon file is checked in the previous statement.